protocol-designer/cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress'
+
+module.exports = defineConfig({
+  video: false,
+  viewportWidth: 1440,
+  viewportHeight: 900,
```

Comment:
```suggestion
  viewportHeight: 1024,
```
i recommend use 1024 instead of 900 since that is the design's base viewport size.

Reply:
agreed.